release-schema.json

```diff
@@ -50,10 +50,38 @@
           "maximum": 1,
           "minimum": 0
         },
+        "competitiveMaximumPercentage": {
+          "title": "Maximum percentage of total value for competitive subcontracting",
```

Comment:
Should it be "contract value" instead of "total value"? Same below.

Reply:
I meant total contract value, but indeed contract value is clearer.

Reply:
Done